### ISSUE TITLE
Suggestion add file to search for completions

### ISSUE THREAD

**Issue by MonkeyToiletLadder** [alienhard/SublimeAllAutocomplete#60]
is there a way to manually add files or directories to the search.  I want auto completion for SDL graphics lib and respective header files, but I don't want to have to open every header file.

**Comment by reagle** [comment]
I agree! I have a big list of Wikipedia templates in a faile and would love to autocomplete from those.
